tests/components/plugwise/test_number.py

```diff
@@ -101,3 +105,30 @@ async def test_adam_temperature_offset_change(
     mock_smile_adam.set_number.assert_called_with(
         "6a3bf693d05e48e0b460c815a4fdd09d", "temperature_offset", 1.0
     )
+
+
+async def test_adam_temperature_offset_out_of_bounds_change(
+    hass: HomeAssistant, mock_smile_adam: MagicMock, init_integration: MockConfigEntry
+) -> None:
+    """Test changing of the temperature_offset number beyond limits."""
+    with pytest.raises(ServiceValidationError):
+        await hass.services.async_call(
+            NUMBER_DOMAIN,
+            SERVICE_SET_VALUE,
+            {
+                ATTR_ENTITY_ID: "number.zone_thermostat_jessie_temperature_offset",
+                ATTR_VALUE: 3.0,
+            },
+            blocking=True,
+        )
+
+    with pytest.raises(MultipleInvalid):
+        await hass.services.async_call(
+            NUMBER_DOMAIN,
+            SERVICE_SET_VALUE,
+            {
+                ATTR_ENTITY_ID: "number.zone_thermostat_jessie_temperature_offset",
+                ATTR_VALUE: "Lower",
+            },
+            blocking=True,
+        )
```

Comment:
This is tested by default I think?

Reply:
Didn't spot it in `number`s tests, but I'll gladly remove it from out test-set?

Reply:
I mean more, this is already tested by HA number code right?

Reply:
Removed as discussed on chat